lib/screens/account.dart

```diff
@@ -0,0 +1,170 @@
+              style: TextStyle(fontWeight: FontWeight.w600),
+            ),
+            onTap: () {},
+          ),
+          ListTile(
+            title: Text(
+              "Change Email",
+              style: TextStyle(fontWeight: FontWeight.w600),
+            ),
+            onTap: () {
+              navigate_option("email_edit", context, model);
+            },
+          ),
+          ListTile(
+            title: Text(
+              "Change Password",
+              style: TextStyle(fontWeight: FontWeight.w600),
+            ),
+          ),
+          logOutButton()
+        ],
+      );
+    });
+  }
+
+  Widget logOutButton() {
+    return ScopedModelDescendant(
+      builder: (BuildContext context, Widget child, MainModel model) {
+        if (model.isAuthenticated) {
+          return ListTile(
+            title: Text(
+              'Sign Out',
+              style: TextStyle(color: Colors.red, fontWeight: FontWeight.w600),
+            ),
+            onTap: () {
+              _showDialog(context, model);
+            },
+          );
+        } else {
+          return Container();
+        }
+      },
+    );
+  }
+
+  void _showDialog(context, model) {
+    showDialog(
+        context: context,
+        builder: (BuildContext context) {
+          return AlertDialog(
+            title: Text("Sign Out"),
+            content: new Text("Are you sure you want to sign out?"),
+            actions: <Widget>[
+              new FlatButton(
+                child: Text(
+                  "Cancel",
+                  style: TextStyle(color: Colors.black),
+                ),
+                onPressed: () {
+                  Navigator.of(context).pop();
+                },
+              ),
+              new FlatButton(
+                child: Text(
+                  "OK",
+                  style: TextStyle(color: Colors.black),
+                ),
+                onPressed: () {
+                  logoutUser(context, model);
+                },
+              )
+            ],
+          );
+        });
+  }
+
+  logoutUser(BuildContext context, MainModel model) async {
+    final SharedPreferences prefs = await SharedPreferences.getInstance();
+    String user_id = prefs.getInt('id').toString();
+    String api_key = prefs.getString('spreeApiKey');
+    Map<String, String> headers = {
+      'Content-Type': 'application/json',
+      'token-type': 'Bearer',
+      'ng-api': 'true',
+      'Auth-Token': api_key,
+      'uid': user_id
+    };
+    http
+        .get(Settings.SERVER_URL + 'logout.json', headers: headers)
+        .then((response) {
+      prefs.clear();
+      model.loggedInUser();
+      model.fetchCurrentOrder();
+    });
+    Navigator.popUntil(
+        context, ModalRoute.withName(Navigator.defaultRouteName));
+  }
+
+  void navigate_option(String screen, BuildContext context, MainModel model) {
+    switch (screen) {
+      case "order_history":
+        {
+          if (model.isAuthenticated) {
```

Comment:
Since, the Account Screen loads only when user is Authenticated, so this check can be avoided here and in subsequent cases.